### ISSUE TITLE
Toggling buffer window with Bufstop command

### ISSUE THREAD

**Issue by muratkarakus7** [mihaifm/bufstop#14]
Hi,

When I use `Bufstop` and then `ESC` keys as a  sequence, I can return back to the same window. 

However, when I use `Bufstop` and then `Bufstop` keys as a sequence, buffer window is dismissed but it focuses on wrong window (left window of the window that I was working).

I think, this shouldn't happen because it become so cumbersome to use `Bufstop` and `ESC` together.
I have mapped `Bufstop` to `<shift-u>` and I want to use it as a toggle. It is really fast compare to using `ESC`. I was using this feature in Buffergator plugin previously.

In summary, `Bufstop` should be a toggle and do what `Bufstop + ESC` does.

Looking forward to hearing from you.

Thanks.

**Comment by mihaifm** [comment]
I'm not sure I understand the issue...
Currently the `ESC` key closes the Bufstop window. If you want it to close automatically after you select a buffer (by pressing a Bufstop key) you can use the `:BufstopFast` command. It does exactly what you describe...it spares you of the effort of pressing the `ESC` key.

Regarding the wrong focus problem, I haven't encountered this...the focus should go to the previous window you were using. Perhaps you can post a screenshot of your window layout in vim, so I can try to replicate it.

**Comment by muratkarakus7** [comment]
Hi, mihaifm

I like `:Bufstop` command and I know what `:BufstopFast` does. My point was different. I have illustrated my idea in the below picture. 

![bufstop toggle issue](https://cloud.githubusercontent.com/assets/2128178/10901084/928dcac4-8223-11e5-9349-791500ab75e1.png)

I just want use `:Bufstop` to open and close the bufstop window so that I can only use a single key mapping for this.

**Comment by mihaifm** [comment]
ah, I understand now. Indeed there is a problem when you use the `:Bufstop` command to close the window.

**Comment by muratkarakus7** [comment]
I would appreciate if you can allocate time and resolve this issue.

I also think you should provide a command to dismiss window, otherwise I have to use `ESC` which is hard to reach and use. However, I should be able to map a custom key mapping to this operation.

**Comment by mihaifm** [comment]
Actally there is an config option fot that, I totally forgot. Add this to your vimrc, it should do the trick:

   let g:BufstopDismissKey="U"

It doesn't work with key combos, like `<leader>u`

**Comment by muratkarakus7** [comment]
Great, it solved :+1: 

I think, you should mention about this configuration in the documentation. People will really need it.

Thank you very much for your effort and time.
